ui/pages/permissions-connect/redirect/index.scss

```diff
@@ -31,7 +31,7 @@
   &__check {
     width: 40px;
     height: 40px;
-    background: white url("/images/permissions-check.svg") no-repeat;
+    background: var(--color-background-default) url("/images/permissions-check.svg") no-repeat;
```

Comment:
Can I be a pain and request this instance be updated to use font-awesome icon. So remove this
```suggestion
```

And updated this instance
![Screen Shot 2022-03-21 at 10 43 46 AM](https://user-images.githubusercontent.com/8112138/159332785-5da05375-3c93-48a5-a70a-1bf4fbc7ceb1.png)

To 
```
<i className="fa fa-check permissions-redirect__check" />
```

You may need to update the color and the font-size in the css to match what it currently looks like

Reply:
Yes, will update :)